docs/features/explorer.md

```diff
@@ -139,6 +139,44 @@ Component.Explorer({
 })
 ```
 
+Here's another example where a map is used to define the order of the **folders** and **files** in the explorer.
+
+```ts title="quartz.layout.ts"
+Component.Explorer({
+  sortFn: (a, b) => {
+    const nameOrderMap: Record<string, number> = {
+      "poetry-folder": 1,
+      "essay-folder": 2,
+      "research-Paper-file": 3,
+      "other-folder": 4,
+    }
+
+    let orderA = 0
+    let orderB = 0
+
+    if (a.file && a.file.slug) {
+      orderA = nameOrderMap[a.file.slug] || 0
+    } else if (a.name) {
+      orderA = nameOrderMap[a.name] || 0
+    }
+
+    if (b.file && b.file.slug) {
+      orderB = nameOrderMap[b.file.slug] || 0
+    } else if (b.name) {
+      orderB = nameOrderMap[b.name] || 0
+    }
+
+    return orderA - orderB
+  },
+})
+```
+
+Depending on your situation, you might have to use `.name`, `.displayName` or `.name.slug`
+
+- `.name` = The default file/folder name in the explorer
+- `.displayName` = The new name you mapped a file/folder name to inside of the explorer
+- `.file.slug` = The raw file/folder name
+
```

Comment:
```suggestion
Instead of `Filenode.file.slug`, you could also change the example to use `Filenode.name`, if you want to use frontmatter titles (or file name if not set) inside `nameOrderMap` instead.

To make this change, swap every occurrence of `a.file.slug`/`b.file.slug` to `a.name`/`b.name` and use the frontmatter title/default file or folder name inside `nameOrderMap`
```

I don't think we should recommend using `.displayName` in official docs as those are only used for display inside the explorer and shouldn't really be used in something like a sort function (also could have side effects if people use `sort` before they use `map`)

Also, im not sure if the existing usage of `a.name`/`b.name` would need to be changed in your example if you were to use `Filenode.name` instead of `Filenode.file.slug`, would be nice if you could verify it working as intended with that change.

Reply:
My current setup looks like this, it might be worth mentioning, that one should give the folders numbers like 100, 200, 300, ... so that the numbers between that can be used to sort the files in the folders without having to constantly rename the order values
![image](https://github.com/jackyzha0/quartz/assets/85735034/d5fb72c7-cead-4320-a2cb-289a97027ac0)

Reply:
just tested it and having the `mapFn` function before or after `sortFn` doesn't matter